lib/testing_appserver/data/user_data/default_user_data.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module TestingAppServer
+  # Module with default user names values
+  module DefaultUserData
```

Comment:
I think those values better to store inside `class UserData` and do not have this module `DefaultUserData` at all